doc/ext/autosummary.rst

```diff
@@ -194,7 +194,8 @@ The following variables available in the templates:
 
 .. data:: underline
 
-   A string containing ``len(full_name) * '='``.
+   A string containing ``len(full_name) * '='``. Use the ``underline`` filter
+   instead.
```

Comment:
This has to stay here for backwards-compatibility, but it's useless when using `| escape`